test/blackbox/common/RTPSWithRegistrationWriter.hpp

```diff
@@ -416,16 +416,6 @@ class RTPSWithRegistrationWriter
         return *this;
     }
 
-    RTPSWithRegistrationWriter& add_throughput_controller_descriptor_to_pparams(
-            uint32_t bytesPerPeriod,
-            uint32_t periodInMs)
-    {
-        eprosima::fastrtps::rtps::ThroughputControllerDescriptor descriptor {bytesPerPeriod, periodInMs};
-        writer_attr_.throughputController = descriptor;
-
-        return *this;
-    }
-
```

Comment:
Is this supposed to be updated with the flow controller method, instead of just deleting it?

Reply:
Of course. Good catch !

Reply:
The method was defined downwards in the file, I force pushed removing the changes in that file